src/rest-server/routes/gateway.router.ts

```diff
@@ -0,0 +1,9 @@
+import express from "express";
+import { getGateway, removeGateway } from "../controllers/gateway.controller";
+
+const gatewayRouter = express.Router();
+
+gatewayRouter.get("/:apiName", getGateway);
```

Comment:
```suggestion
gatewayRouter.use("/:apiName", dispatcher);
```
Este endpoint no debería ser un .get, sino un .use() porque imaginate que te mandan un post, ya no lo recibirías.
El controller en el futuro va a ser una clase la cual va a recibir por inyección el servicio que se comunique con el load balancer, pero aparte el nombre del metodo (en este caso getGateway) debería ser otro ya que en realidad vos no le estas pidiendo al endponit que te devuelva un gateway. Podria ser dispatcher por ejemplo